migrate/migrate.go

```diff
@@ -61,6 +67,13 @@ func (m *Migrator) MigrateProject(id uuid.UUID, from, to *semver.Version) (bool,
 		}
 	}
 
+	if from.LessThan(V0_8_0) {
```

Comment:
Maybe this code could re refactored to something like this:

you define a migration interface that has `Version() *semver.Version` and `Migrate(m *Migrator, id uuid.UUID)`
You then put these in an ordered list by `Version` and for each where version < current version run Migrate.

This way it will be easier to add new migrations in the future and there will be a single error handling location.